internal/pkg/logs/service.go

```diff
@@ -0,0 +1,85 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package logs
+
+import (
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/aws/copilot-cli/internal/pkg/aws/cloudwatchlogs"
+	"github.com/aws/copilot-cli/internal/pkg/aws/sessions"
+	"github.com/aws/copilot-cli/internal/pkg/config"
+	"github.com/aws/copilot-cli/internal/pkg/term/log"
+)
+
+const (
+	logGroupNamePattern = "/copilot/%s-%s-%s"
+)
+
+type logGetter interface {
+	TaskLogEvents(logGroupName string,
```

Comment:
I think this method needs to be renamed to just `LogEvents` since the "task" concept is not part of cloudwatch and instead is an ECS concept.